README.md

```diff
@@ -103,15 +103,23 @@ Compile / packageAvro / publishArtifact := true
 You can specify a dependency on an avro source artifact that contains the schemas like so:
 
 ```sbt
-libraryDependencies += "org" % "name" % "rev" classifier "avro"
+libraryDependencies += "org" % "name" % "rev" % "avro" classifier "avro"
```

Comment:
If people use the previous dependency declaration will the transitively included deps be compiled into the current artifact?

Reply:
there are 2 cases when user update to `sbt-avro` v4 :
- if they require the schemas to be compiled: compilation fails unless user changes the dep scope to `avro` (or update the include filter)
-  if one lib pulls an undesired `avro` classifier artifact, the plugin won't re-compile the schemas (we assume the jar pulling the schema already includes compiled classes)